### ISSUE TITLE
Question about reproducing the result on wmt'14 en-de.

### ISSUE THREAD

**Issue by chenyushuo** [cooelf/UVR-NMT#9]
Firstly, thanks for your great work.

I have followed the steps in README and do not modify any other parameters, but I cannot reproduce the result in your paper. I have noticed the issue #6 and I train this model on 8 GeForce RTX 3090 following your advice. However, I get 27.81 on the test set, which is still lower than the result in your paper. I also test the result of the [model](https://drive.google.com/open?id=1cRwFiT0nWJq2gWecMof8I5Mna3BFnlVx) you published in README, which could only get 27.91 on the test set. Could you give me some advice to reproduce the result in your paper? Thanks!

**Comment by cooelf** [comment]
Yes. The provided model is my reproduced one using 3090 servers, which is slightly lower than the reported one. The best model was originally trained on the V100 server two years ago in our cooperative institute but I could not have the access to the servers anymore. The hyper-parameters should be the same.
